helpers/trees/merkle-tree.ts

```diff
@@ -62,15 +63,15 @@ export default class MerkleTree {
   }
 
   getRoot(): Buffer {
-    return this.layers[this.layers.length - 1][0];
+    return this.layers[this.layers.length - 1]![0]!;
```

Comment:
why '!' ?

Reply:
The eslint of the consumer, specifically the dApp is enforcing this rule. We'll need to re-visit this once we break out the dApp.